### ISSUE TITLE
Default font in editor

### ISSUE THREAD

**Issue by iamchathu** [RainLoop/rainloop-webmail#844]
It's better if we have a feature to set default font family and color in the email body.

**Comment by mikerockett** [comment]
This is something I need as well. Not a fan of Arial in emails (who knows why), but this is defintely something I believe RainLoop should have. Not sure on colour, but definitely font name and size.

**Comment by bms8197** [comment]
I agree. I am a big fan of Verdana. I don't like Arial. Everytime I am writing an email I have to switch the fonts!

**Comment by bpeterman** [comment]
I also agree with this, any status?

**Comment by bogdanstoica35** [comment]
I have tried that. Even modified the font in ckeditor.js but for some reason, every time I start a new message it seems that the ckeditor.js file is regenerated since even if I leave there strictly Verdana font, then all the font families are back again and so on. There should be another way to do this

**Comment by bogdanstoica35** [comment]
So nobody?!

**Comment by tgrignon** [comment]
+1 agree

**Comment by pyheme** [comment]
+1 
Would be a great addition.

**Comment by pyheme** [comment]
Hi, 
Not to revive an old threat but, should someone else land here looking for an answer, I could change the 'default font' in: 
`./rainloop/v/[rainloop.version.number]/static/js/min/app.min.js` 
(v/1.12.1 at the time of this writing).
Just look for the string `Arial` and change to the font you'd like. 

Interestingly, this edit changes the font in the code of the email but not in the wysiwyg editor.  For this to change too, it appears that you need to edit as well the class `.cke_wysiwyg_div` in `./rainloop/v/[rainloop.version.number]/static/css/app.min.css` 

Finally, if you'd like to also change the order of the fonts in the drop down list (or add your favorite font on top), that can be done looking for the string `CKEDITOR.config.font_names` in the file `./rainloop/v/[rainloop.version.number]/static/ckeditor/ckeditor.js`

Note that it works but is a poor fix as all these changes are in the ./rainloop/v/[rainloop.version.number] folder and, therefore, get lost on every new update of rainloop.

**Comment by miguelbustosnet** [comment]
The solution provided by @pyheme is great for composing e-mails. But it does not work on e-mail content. Although when writing you can see the text in the font and size according to the new settings, the e-mail is sent with the original ones.

What would be interesting is that people see the same font family and size that we want.

Try mailing yourself to see what I mean!

**Comment by pyheme** [comment]
Hi @miguelbustosnet 
Which version of Rainloop are you running?  Just did a test sending myself an email to the same email and to my gmail address and both came in with the proper font.
I'm using Rainloop v1.12.1 Community edition and using Chrome as browser.  Could it be that your setup is different and that the .js files you're loading are different ones?

If you try looking for all files containing the word Arial in your install folder, you can rather easily identify which ones to change... maybe with a bit of trial and errors. Just make sure to make copies if not sure what you're doing. 
(In Ubuntu, the command `grep -rl /path/to/your/rainloop/install/ -e 'Arial'` will output a list of files containing the word Arial in your rainloop install. Not too familiar with other distribs. but I'd assume it should work the same).

**Comment by miguelbustosnet** [comment]
Thanks for your reply @pyheme.

I've just taken a look into the app.min.css and found that I changed the font size in  `font_defaultLabel:"Arial",fontSize_defaultLabel:"13"`, when the correct place is the following Arial search match `<div data-html-editor-font-wrapper="true" style="font-family: arial, sans-serif; font-size: 13px;">`.

I've also modified the `body {\n\tbackground-color: #fff;\n\tfont-size: 13px;\n\tfont-family: arial, sans-serif;\n\tmargin: 0;\n\tpadding: 0;\n` piece of code to make sure that changes are effective.

I've checked it after doing those modifications and it works excellently! Using v1.12.1.

**Comment by StefanGebhardt** [comment]
+1
A function in the gui would be great and not after an update editing css-files

**Comment by joern45219** [comment]
Hey guys, 

have looked for the folder mentioned above in my nextcloud instance running rainloop. Did not find this one. Any idea where this hides? 

Also is it possible to add any webfonts into this instead of the regular ones like Arial / Verdana? 

Thanks already
Joern

**Comment by GP1247** [comment]
> Hey guys,
> 
> have looked for the folder mentioned above in my nextcloud instance running rainloop. Did not find this one. Any idea where this hides?
> 
> Also is it possible to add any webfonts into this instead of the regular ones like Arial / Verdana?
> 
> Thanks already Joern

+1 on this.
I'm also limited to only four fonts on RainLoop on a NextCloud service.
